File: tests-e2e/login.test.ts
import { test, expect } from '@playwright/test';

test('Login correcto', async ({ page }) => {

  await page.goto('https://ezequiel-moran-tp-2-prog-4-2026-fro.vercel.app');

  await page.fill('#usuario', 'Palandri01');

  await page.fill(
    'input[formControlName="password"]',
    'Palandri01'
  );

  await page.click('button.btn-primary');

  await expect(page).toHaveURL(/publicaciones/);

  await page.waitForTimeout(5000); // mantiene abierta la página 5 segundos

})

File: tests-e2e/login_test.py
from playwright.sync_api import sync_playwright, expect
import re


def test_login():

    with sync_playwright() as p:

        browser = p.chromium.launch(headless=False)

        page = browser.new_page()

        page.goto(
            "https://ezequiel-moran-tp-2-prog-4-2026-fro.vercel.app"
        )

        page.fill(
            "#usuario",
            "Palandri01"
        )

        page.fill(
            'input[formControlName="password"]',
            "Palandri01"
        )

        page.click(
            "button.btn-primary"
        )

        expect(page).to_have_url(
            re.compile(".*publicaciones.*")
        )

        page.wait_for_timeout(5000)

        browser.close()


test_login()